deb/openmediavault/workbench/src/index.html

```diff
@@ -37,6 +38,8 @@
       animation: toggle-opacity 1s step-start infinite;
     }
   </style>
+  <link rel="manifest" href="manifest.webmanifest">
+  <meta name="theme-color" content="#1976d2">
```

Comment:
The [corporate color](https://github.com/openmediavault/openmediavault/blob/d97586361c7fb9c0d830d2e281f57dc280fdb9ba/deb/openmediavault/workbench/src/scss/defaults/_colors.scss#L23) is #5dacdf.

```suggestion
  <meta name="theme-color" content="#5dacdf">
```

Reply:
Thanks a lot for the suggested changes. The color has been updated accordingly 👍